roseau/load_flow/models/loads/loads.py

```diff
@@ -85,6 +85,14 @@ def is_flexible(self) -> bool:
         """Whether the load is flexible or not. Only :class:`PowerLoad` can be flexible."""
         return False
 
+    @property
+    def voltage_phases(self) -> list[str]:
+        """The phases of the load voltages."""
+        if "n" in self.phases:  # "an", "bn", "cn"
+            return [p + "n" for p in self.phases[:-1]]
+        else:  # "ab", "bc", "ca"
+            return [p1 + p2 for p1, p2 in zip(self.phases, np.roll(list(self.phases), -1))]
+
```

Comment:
I don't think it works for 2-phases delta loads, it should only return `"ab"` and not `"ba"`.